hinted-compilation/get-hints.py

```diff
@@ -360,6 +360,17 @@ def myexit(lname, jname, trace_logic):
 # -----------------------------------------------------------------------------
 # Main program
 # -----------------------------------------------------------------------------
+timeout = 5 * 60 # default timeout 5 minutes
+if sys.argv[1] == "--timeout":
+    try:
+        timeout = int(sys.argv[2])
+        if timeout <= 0: timeout = None
+        else: timeout *= 60
+        del sys.argv[1:3]
+    except:
+        sys.exit("Invalid timeout value (specify positive integer for timeout in minutes, use 0 to turn off timeout)")
+if timeout and timeout > 0: print(f"Process run will timeout in {timeout//60} minutes")
```

Comment:
Unfortunately f'' strings are not allowed in Nuitka yet.

Reply:
Replaced with regular formatting. Is this because of a Python version limitation?

Reply:
Yes sorry, but we continue to support before Python 3.6.